### PR TITLE
bump aiida-prerequisites

### PR DESCRIPTION
**Pull request by ltalirz**
This includes an upgrade from Ubuntu 18.04 to 20.04.

Also remove installation of custom circus dependency (issue was fixed in
circus 17.0).

### CODE REVIEW

File: .jenkins/Jenkinsfile [hunk 1]
```diff
@@ -48,10 +48,10 @@ pipeline {
                     // this folder is mounted from a volume so will have wrong permissions
                     sh 'sudo chown root:root /home/jenkins/.cache'
                     // prepare environment (install python dependencies etc)
-                    sh 'pip install -r requirements/requirements-py-3.7.txt --cache-dir /home/jenkins/.cache/pip'
+                    sh 'pip install -r requirements/requirements-py-3.8.txt --cache-dir /home/jenkins/.cache/pip'
```
Comment (csadorf) [inline .jenkins/Jenkinsfile:51]: Why 3.8 and not 3.9?

**Review by csadorf — CHANGES_REQUESTED** [review]
There are a lot of checks failing and it looks like the change of the Python version requires an update of the Jenkins CI job.

File: .jenkins/Jenkinsfile (hunk 1 above)
Comment (ltalirz) [reply .jenkins/Jenkinsfile:51]: I think it has to be the native python version (which is 3.8 for ubuntu 20.04)

Comment (ltalirz) [reply .jenkins/Jenkinsfile:51]: > There are a lot of checks failing and it looks like the change of the Python version requires an update of the Jenkins CI job.

Ehm... are you looking at the first or the second commit?

Comment (csadorf) [reply .jenkins/Jenkinsfile:51]: Indeed, I looked at the first commit for the jenkins failure, but there are still these "failed check" annotations, do you know what this is about?
Screen Shot 2021-09-14 at 09 28 28

Comment (ltalirz) [reply .jenkins/Jenkinsfile:51]: I don't know - when I click on the icons in the right top corners it brings to jobs that succeeded just fine.

Also, these tests are unrelated to the dockerfile.

I think it's somehow a leftover from the first commit

Comment (csadorf) [reply .jenkins/Jenkinsfile:51]: Ok, I assume it is fine then.

**Review by csadorf — APPROVED** [review]

**Pull request merged**